diff -up a/Cargo.toml.orig b/Cargo.toml
--- a/Cargo.toml.orig	2025-06-19 18:18:08.225290250 +0200
+++ b/Cargo.toml	2025-06-19 18:20:29.142344871 +0200
@@ -23,6 +23,12 @@ categories = ["cryptography", "authentic
 license = "LGPL-2.0-or-later"
 edition = "2021"
 rust-version = "1.79"
+exclude = [
+    "/deny.toml",
+    "/Dockerfile",
+    "/sq.subplot",
+    "/sq-subplot.md",
+]
 
 [badges]
 gitlab = { repository = "sequoia-pgp/sequoia-sq" }
@@ -66,11 +72,6 @@ termcolor = "1.2.0"
 textwrap = { version = ">=0.15, <0.17", default-features = false, features = ["smawk", "unicode-width"] }
 typenum = "1"
 
-# This is for subplot only.  If you don't enable the subplot feature,
-# feel free to patch it out.
-subplotlib = { version = ">=0.11, <0.13", optional = true }
-culpa = { version = "1", optional = true }
-
 [build-dependencies]
 anyhow = "1.0.18"
 buffered-reader = { version = "1.4", default-features = false, features = ["compression"] }
@@ -90,10 +91,6 @@ typenum = "1"
 cfg-if = "1"
 terminal_size = ">=0.2.6, <0.5"
 
-# This is for subplot only.  If you don't enable the subplot feature,
-# feel free to patch it out.
-subplot-build = { version = ">=0.11, <0.13", optional = true }
-
 [dev-dependencies]
 assert_cmd = "2"
 predicates = ">=2, <4"
@@ -105,22 +102,13 @@ name = "sq"
 path = "src/main.rs"
 bench = false
 
-[[test]]
-name = "sq-subplot"
-path = "tests/sq-subplot.rs"
-required-features = ["subplot"]
-
 [features]
 default = [
-    "crypto-nettle",
+    "crypto-openssl",
 ]
 crypto-nettle = ["sequoia-openpgp/crypto-nettle"]
 crypto-openssl = ["sequoia-openpgp/crypto-openssl"]
-crypto-botan = ["sequoia-openpgp/crypto-botan"]
-crypto-botan2 = ["sequoia-openpgp/crypto-botan2"]
-crypto-cng = ["sequoia-openpgp/crypto-cng"]
 crypto-rust = ["sequoia-openpgp/crypto-rust"]
-subplot = ["culpa", "subplot-build", "subplotlib"]
 
 [profile.release]
 debug = true
